pkgs/development/python-modules/rapidocr/default.nix

```diff
@@ -0,0 +1,158 @@
+backendId:
```

Comment:
I don't think this is allowed in `nixpkgs`.

Reply:
Perhaps, or just don't supply a default to make it fail if it's not called via callPackage with the right arg

Reply:
@natsukium What do you recommend here?

Reply:
We should have one file per package.
The python library is often broken, so we set attributes like `disabled` and `meta.broken`, and I don't want to put another condition, backend. `paddlepaddle` is an example.

Monorepo packages, for example `opentelemetry`, could share the same version, src, etc.
It may be more duplicated than the current programmatic one, and it may be uglier, depending on your preference, but for future maintenance, this is the way to go.
https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/development/python-modules/opentelemetry-api/default.nix
https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/development/python-modules/opentelemetry-sdk/default.nix

Reply:
Honestly the approach showed above isn't very DRY and requires a lot of duplicated code, or sourcing code from one place only to change it later. Since 95% of the derivation is exactly the same across all backends, I find that approach adds a needless burden on maintainers who have to make changes to all three runtimes once upstream changes something.

Reply:
How about merging `rapidocr-onnxruntime` for the moment which seems to be in good state, and decide what to do after that? I'm badly looking for that package in `open-webui` actually.

Reply:
Sure!